packages/hash/frontend/src/shared/ui/theme/components/data-display/mui-chip-theme-options.tsx

```diff
@@ -0,0 +1,179 @@
+import { faClose } from "@fortawesome/free-solid-svg-icons";
+import { ChipProps, Components, PaletteValue, Theme } from "@mui/material";
+import { FontAwesomeIcon } from "../../../../icons";
+
+const getColors = (
+  theme: Theme,
+  color: Exclude<ChipProps["color"], "default" | undefined> = "gray",
+  variant: Exclude<ChipProps["variant"], undefined> = "filled",
+) => {
+  let textColorShade = 80;
+  let bgColorShade = 20;
+  let textHoverColorShade = 90;
+  let bgHoverColorShade = 30;
+  let outlineColorShade: number | undefined = undefined;
+  let outlineHoverColorShade: number | undefined = undefined;
+
+  // For most colors, on hover, we increase the shade
+  // of text color and bg color by 10
+  switch (color) {
+    case "yellow":
+      bgColorShade = 30;
+      bgHoverColorShade = bgColorShade + 10;
+      textColorShade = 90;
+      textHoverColorShade = textColorShade + 10;
+      break;
+    case "navy":
+      bgColorShade = 20;
+      bgHoverColorShade = bgColorShade + 10;
+      textColorShade = 70;
+      textHoverColorShade = 90;
+      break;
+    case "red":
+    case "pink":
+      bgColorShade = 20;
+      bgHoverColorShade = bgColorShade + 10;
+      textColorShade = 90;
+      textHoverColorShade = textColorShade + 10;
+      break;
+    case "blue":
+    case "purple":
+    case "orange":
+    case "mint":
+    case "teal":
+    default:
+      bgColorShade = 20;
+      bgHoverColorShade = bgColorShade + 10;
+      textColorShade = 80;
+      textHoverColorShade = textColorShade + 10;
+      break;
+  }
+
+  // if it's an outlined variant,
+  // 1. we set outline color to initial bgColorShade + 10
+  // 2. we set outline hover color to outline color (set above) + 10
+  // 2. we reduce all other shades by 10
```

Comment:
```suggestion
  // 3. we reduce all other shades by 10
```

🚲⛺️-ing 😅

Reply:
Thanks for catching. Fixed in [289337a](https://github.com/hashintel/hash/pull/495/commits/289337a658b31c940cbc622b0e239600141eceed)